testbed/HIL_Firmware/common/data/HIL.dbc

```diff
@@ -36,6 +36,20 @@ BS_:
 BU_: Computer PDU_HIL VCU_HIL
 
 
+BO_ 2281901827 Relay_PDU_outputs: 2 PDU_HIL
+ SG_ Pow_right_fan : 11|1@1+ (1,0) [0|1] ""  Computer
```

Comment:
This is a nitpick but in our car DBC we only use _ to separate a board name from the signal name and the words in the signal name is all caps. See the DBC for examples

Reply:
will fix